cryptographic_estimators/MQEstimator/MQAlgorithms/bjorklund.py

```diff
@@ -71,20 +74,41 @@ def lambda_(self):
             sage: E.lambda_()
             3/10
         """
+        
+        time = inf
```

Comment:
@aangulog I think  instead of writting the `_lambda` method it is better to rewrite the `find_optimal_parameters` for the particular case of `bjorklund`. It will solve the issue for the `TildeO` optimal params at the same time.